src/screens/Tasks/Category/CategoryEditModal.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useState } from "react";
+import { Modal, Form, Button } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as Yup from "yup";
+import { useDispatch } from "react-redux";
+import { useToast } from "~/components/Toast";
+import { Category } from "~/types/Category";
+import { UpdateCategoryRequest } from "~/services/categoryApi";
+import { updateCategory, getCategories } from "~/store/slices/categorySlice";
+import { AppDispatch } from "~/store/store";
+
+interface CategoryEditModalProps {
+	isOpen: boolean;
+	category: Category;
+	onClose: () => void;
+}
+
+const categorySchema = Yup.object().shape({
+	name: Yup.string()
+		.trim()
+		.required("Tên danh mục không được để trống")
+		.min(2, "Tên danh mục phải có ít nhất 2 ký tự"),
+	description: Yup.string()
+		.optional()
+		.max(500, "Mô tả không được vượt quá 500 ký tự"),
+});
+
+const CategoryEditModal: React.FC<CategoryEditModalProps> = ({
+	isOpen,
+	category,
+	onClose,
+}) => {
+	const dispatch = useDispatch<AppDispatch>();
+	const { showToast } = useToast();
+	const {
+		register,
+		handleSubmit,
+		setValue,
+		formState: { errors, isValid },
+	} = useForm({
+		resolver: yupResolver(categorySchema),
+		mode: "onChange",
+		defaultValues: { name: "", description: "" },
```

Comment:
Bỏ phần setValue bên dưới đi em nhá
```suggestion
		defaultValues: { name: category.name, description: category.description },
```